django-backend/fecfiler/authentication/views.py

```diff
@@ -140,11 +147,10 @@ def login_redirect(request):
 
 @api_view(["GET"])
 @require_http_methods(["GET"])
+@permission_classes([])
```

Comment:
I'm wondering if we need @authentication_classes([]) here as well. When testing authenticate_login() and authenticate_logout() functions, they were getting authentication denied responses without that decorator.

Reply:
@mjtravers yes I wondered about this too, however I think since we are no longer using 
`rest_framework_jwt.authentication.JSONWebTokenAuthentication` in our `DEFAULT_AUTHENTICATION_CLASSES` that this is no longer needed.  I tested without it and it seems to work, so I opted to leave it out.

Reply:
Sounds good.